meshtastic/stream_interface.py

```diff
@@ -124,6 +125,23 @@ def close(self):
         if self._rxThread != threading.current_thread():
             self._rxThread.join()  # wait for it to exit
 
+    def _handleLogByte(self, b):
+        """Handle a byte that is part of a log message from the device."""
+
+        utf = "?"  # assume we might fail
+        try:
+            utf = b.decode("utf-8")
+        except:
+            pass
+
+        if utf == "\r":
+            pass    # ignore
+        elif utf == "\n":
+            self._handleLogLine(self.cur_log_line)
```

Comment:
I guess this is where we're handling log stuff that doesn't have a full LogRecord attached to it. I guess maybe the thing to do will just be to construct one with an "unset" log level. Over time more of this will move into proper protobufs anyway, I suppose